WordPress/build.gradle

```diff
@@ -51,7 +50,6 @@ android {
 
     productFlavors {
         vanilla { // used for release and beta
-            buildConfigField "boolean", "AZTEC_EDITOR_AVAILABLE", "false"
```

Comment:
I was going to do that in `develop` after the release. I just wanted to make it work for now. But I can do it all here if you prefer.